templates/frontend/objects/article_summary.tpl

```diff
@@ -22,15 +22,15 @@
 <div class="article-summary media">
 	{if $article->getLocalizedCoverImage()}
 		<div class="cover media-left">
-			<a href="{url page="article" op="view" path=$articlePath}" class="file">
+			<a href="{url journal=$journal page="article" op="view" path=$articlePath}" class="file">
 				<img class="media-object" src="{$article->getLocalizedCoverImageUrl()|escape}" alt="{$article->getLocalizedCoverImageAltText()|escape|default:''}">
 			</a>
 		</div>
 	{/if}
 
 	<div class="media-body">
 		<h3 class="media-heading">
-			<a href="{url page="article" op="view" path=$articlePath}">
+			<a href="{url journal=$journal page="article" op="view" path=$articlePath}">
```

Comment:
Does this work? I think `journal=$journal` needs to be `journal=$journal->getPath()` to match what we're doing in the default theme.

Reply:
Its the only way it does work.
With journal=$journal->getPath() the issue remains.

Reply:
Hmm, there must be something odd going on. Our default theme uses `$journal->getPath()` and it also includes an `{if}` statement so that the parameter is only added when `$journal` exists. I just tested this out and it's working in the default theme.

What version of OJS are you using?

Reply:
3.2.1 the latest one

Reply:
Ahh, I see what's happening. I missed the part in your changes where you're passing in the path to the template:

https://github.com/NateWr/bootstrap3/pull/155/files#diff-9c64ea7f12c9669270ecddc690ff9bf0R90

Instead of doing it that way, let's pass in the full journal object to the template. That way we'll stay consistent with how it's done in other themes. So in `search.tpl` change this:

```
journal=$result.journal->getPath()
```

To this:

```
journal=$result.journal
```

Then in `article_summary.tpl` we can pass the path in directly. However, we need to check if the `$journal` exists so that this doesn't break the article summary template if it is used without passing in the `$journal` variable. So let's make it:

```
<a href="{if $journal}{url journal=$journal->getPath() page="article" op="view" path=$articlePath}{else}{url page="article" op="view" path=$articlePath}{/if}">
```

You'll also need to make the same change to the cover image URL (line 25 of `article_summary.tpl`).

Reply:
Yeap, it looks like its working like that. thank you